Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/core.py", line 477, in signal_shutdown
    h(reason)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/registration.py", line 376, in cleanup
    (errno, msg) = se.args
ValueError: need more than 1 value to unpack
Exception in thread Thread-3 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
  File "/usr/lib/python2.7/threading.py", line 754, in run
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 154, in run
<type 'exceptions.AttributeError'>: 'NoneType' object has no attribute 'timeout'
